src/game.cpp

```diff
@@ -3535,7 +3535,6 @@ void Game::processClientEvents(CameraOrientation *cam, float *damage_flash)
 		} else if (event.type == CE_HUDADD) {
 			u32 id = event.hudadd.id;
 
-			LocalPlayer *player = client->getEnv().getLocalPlayer();
```

Comment:
When i looked at this function i see we call this function here but it was already called in the parent scope.